addons/mail_group/models/mail_group.py

```diff
@@ -39,7 +39,7 @@ class MailGroup(models.Model):
     @api.model
     def default_get(self, fields):
         res = super(MailGroup, self).default_get(fields)
-        if not res.get('alias_contact') and (not fields or 'alias_contact' in fields):
+        if not res.get('alias_contact') and 'alias_contact' in fields:
```

Comment:
Nitpicking, feel free to ignore
```suggestion
        if 'alias_contact' in fields and not res['alias_contact']:
```